x/wasm/keeper/msg_server_integration_test.go

```diff
@@ -406,30 +456,14 @@ func TestExecuteContract(t *testing.T) {
 			}
 
 			// check event
-			events := rsp.Events
-			assert.Equal(t, 4, len(events))
-			assert.Equal(t, "message", events[0].Type)
-			assert.Equal(t, 2, len(events[0].Attributes))
-			assert.Equal(t, "module", string(events[0].Attributes[0].Key))
-			assert.Equal(t, "wasm", string(events[0].Attributes[0].Value))
-			assert.Equal(t, "sender", string(events[0].Attributes[1].Key))
-			assert.Equal(t, myAddress.String(), string(events[0].Attributes[1].Value))
-			assert.Equal(t, "execute", events[1].Type)
-			assert.Equal(t, 1, len(events[1].Attributes))
-			assert.Equal(t, "_contract_address", string(events[1].Attributes[0].Key))
-			assert.Equal(t, instantiateResponse.Address, string(events[1].Attributes[0].Value))
-			assert.Equal(t, "wasm", events[2].Type)
-			assert.Equal(t, 3, len(events[2].Attributes))
-			assert.Equal(t, "_contract_address", string(events[2].Attributes[0].Key))
-			assert.Equal(t, instantiateResponse.Address, string(events[2].Attributes[0].Value))
-			assert.Equal(t, "action", string(events[2].Attributes[1].Key))
-			assert.Equal(t, "release", string(events[2].Attributes[1].Value))
-			assert.Equal(t, "destination", string(events[2].Attributes[2].Key))
-			assert.Equal(t, "wasm-hackatom", events[3].Type)
-			assert.Equal(t, "_contract_address", string(events[3].Attributes[0].Key))
-			assert.Equal(t, instantiateResponse.Address, string(events[3].Attributes[0].Value))
-			assert.Equal(t, "action", string(events[3].Attributes[1].Key))
-			assert.Equal(t, "release", string(events[3].Attributes[1].Value))
+			assert.Equal(t, len(spec.expEvents), len(rsp.Events))
+			assert.Equal(t, spec.expEvents[0], rsp.Events[0])
+			assert.Equal(t, spec.expEvents[1], rsp.Events[1])
+			assert.Equal(t, spec.expEvents[2].Attributes[2].Key, rsp.Events[2].Attributes[2].Key)
+			// Note: Value with destination as key cannot be tested because it is a different value for each execution
+			assert.Equal(t, spec.expEvents[2].Attributes[0], rsp.Events[2].Attributes[0])
+			assert.Equal(t, spec.expEvents[2].Attributes[1], rsp.Events[2].Attributes[1])
+			assert.Equal(t, spec.expEvents[3], rsp.Events[3])
```

Comment:
```suggestion
			new_spec := spec
			// Note: Value with destination as key cannot be tested because it is a different value for each execution
			new_spec.expEvents[2].Attributes[2].Value = rsp.Events[2].Attributes[2].Value
			assert.Equal(t, new_spec.expEvents, rsp.Events)
```

Reply:
Thanks. Since the commit history has changed, I will adopt this change here

Reply:
Since the commit history has changed, I will adopt this change here